docs/wiki/configurator/failsafe-tab.md

```diff
@@ -4,8 +4,125 @@ sidebar_position: 5
+| **Maximum pitch angle**                 | The maximum pitch angle the craft will use to return to the home position. Rescue throttle usually needs to be increased if the pitch angle is increased. If pitch angle is too low, the craft might not be able to return in high winds.                                                                                                                                                                                                         |
+| **Descent rate (meters/second)**        | The rate at which the craft will descend to the ground when it reaches the home position.                                                                                                                                                                                                                                                                                                                                                         |
+| **Throttle minimum**                    | The **minimum** throttle value the craft will use to return to the home position.                                                                                                                                                                                                                                                                                                                                                                 |
+| **Throttle maximum**                    | The **maximum** throttle value the craft will use to return to the home position.                                                                                                                                                                                                                                                                                                                                                                 |
+| **Throttle hover**                      | The throttle value the craft will use to hover at the return altitude. **It is important to set this value accurately.**                                                                                                                                                                                                                                                                                                                          |
+| **Minimum distance to home (meters)**   | The minimum distance that the craft has to be from the home position to activate GPS Rescue. If the rescue starts too close to home, the craft will fly away until this distance is reached, then start the normal rescue behavior.                                                                                                                                                                                                               |
```

Comment:
The minimum distance that the craft has to be from the home position for a 'clean' GPS Rescue.  If the rescue starts within the minimum distance, the craft will fly away until *at least* this distance is reached, before starting the normal rescue behaviour.  The craft may travel a considerable distance in a random direction before returning to Home, so it is best to avoid initiating within the minimum distance.